data/opam_docs/2.1/03_faq.md

```diff
@@ -0,0 +1,446 @@
+
+Here is an example of an ordered sequence: `~~`, `~`, `~beta2`, `~beta10`, `0.1`,
+`1.0~beta`, `1.0`, `1.0-test`, `1.0.1`, `1.0.10`, `dev`, `trunk`
+
+---
+
+#### 🐫  What does the `--jobs` option do? It doesn't seem to enable parallel builds.
+
+It does, but at the _package_ granularity: it will only be noticeable if you
+build independent packages in the same command. Each package has its own build
+commands, and it's up to them to enable parallelism.
+
+If you are a packager, you may use the `jobs` opam variable, e.g. `make
+"-j%{jobs}%"`.
+
+---
+
+#### 🐫  opam wants to do reinstallations after update. Can I skip them?
+
+When opam detects meaningful changes in upstream packages, it marks them for
+reinstallation, to be sure you get the latest fixes — repository managers
+generally don't abuse modifying existing packages. You can check this with:
+
+```
+opam reinstall --list-pending
+```
+
+And, in case you want to skip them:
+```
+opam reinstall --forget-pending
+```
+
+You should only do this if you know the changes you are skipping, though.
+
+---
+
+#### 🐫  Some package installation fails with "ocamlfind: already installed"
+
+Probably the package was either installed from outside of opam, or uncleanly
+removed. You should try:
+
+```
+opam remove <package> --force
+opam install <package>
+```
+
+This will process the uninstall instructions, even if opam has no knowledge of
+the package being installed. You may also try to uninstall directly with
+ocamlfind, or just remove the problematic files.
+
+---
+
+#### 🐫  opam is slow on top of NFS. How can I make it faster?
+
+opam root is usually located in the `home` directory, which, on top of NFS,
+slows down opam operations. Locating opam root in `/tmp` is not a solution:
+you could lose your opam configuration at each reboot.
+
+You can use the [`nfsopam`](https://github.com/UnixJunkie/nfsopam) script to
+have the best of both worlds: persistence of NFS directories and fast operations
+of local directories.
+
+---
+
+#### 🐫  What does the `--cli` option do? Should I be using it everywhere?
+
+`--cli` was introduced in opam 2.1 to deal with changes in the command line
+interface between releases. It tells opam to interpret the command line as a
+specific version, in particular it means that new options or options which
+have had their meaning altered will not be available, or will be behave as they
+did in that version. It only affects the command-line - it does not, for
+example, stop a root from being upgraded from an older version to the current
+version.
+
+We recommend using it in scripts (and programs which call opam) since they can
+then expect to work seamlessly with future versions of the opam client. It's
+also a good idea to use it in blog posts, or other documentation you may share,
+since it allows copy-and-paste to work reliably (a user with a newer version of
+opam should have no issues and a user with an older opam gets a clearer error
+message).
```

Comment:
```suggestion
since it allows copy-and-paste to work reliably. (A user with a newer version of
opam should have no issues and a user with an older opam gets a clearer error
message.)
```